internal/test/data/vector/gen_test.go

```diff
@@ -752,3 +753,380 @@ func Test_gaussianDistributedUint8VectorGenerator(t *testing.T) {
+						val, val, val, val, val,
+					},
+				},
+				checkFunc: defaultCheckFunc,
+			}
+		}(),
+	}
+
+	for _, tc := range tests {
+		test := tc
+		t.Run(test.name, func(tt *testing.T) {
+			tt.Parallel()
+			defer goleak.VerifyNone(tt, goleak.IgnoreCurrent())
+			if test.beforeFunc != nil {
+				test.beforeFunc(test.args)
+			}
+			if test.afterFunc != nil {
+				defer test.afterFunc(test.args)
+			}
+			checkFunc := test.checkFunc
+			if test.checkFunc == nil {
+				checkFunc = defaultCheckFunc
+			}
+
+			got := GenSameValueVec(test.args.size, test.args.val)
+			if err := checkFunc(test.want, got); err != nil {
+				tt.Errorf("error = %v", err)
+			}
+		})
+	}
+}
+
+func TestConvertVectorsUint8ToFloat32(t *testing.T) {
+	type args struct {
+		vectors [][]uint8
+	}
+	type want struct {
+		wantRet [][]float32
+	}
+	type test struct {
+		name       string
+		args       args
+		want       want
+		checkFunc  func(want, [][]float32) error
+		beforeFunc func(args)
+		afterFunc  func(args)
+	}
+	defaultCheckFunc := func(w want, gotRet [][]float32) error {
+		if !reflect.DeepEqual(gotRet, w.wantRet) {
+			return errors.Errorf("got: \"%#v\",\n\t\t\t\twant: \"%#v\"", gotRet, w.wantRet)
+		}
+		return nil
+	}
+	tests := []test{
+		{
+			name: "convert 1 float32 vector from uint8 vector",
+			args: args{
+				vectors: [][]uint8{
+					{
+						1, 2, 3,
+					},
+				},
+			},
+			want: want{
+				wantRet: [][]float32{
+					{
+						1, 2, 3,
+					},
+				},
+			},
+		},
+		{
+			name: "convert 3 float32 vector from uint8 vector",
+			args: args{
+				vectors: [][]uint8{
+					{
+						1, 2, 3,
+					},
+					{
+						4, 5, 6,
+					},
+					{
+						7, 8, 9,
+					},
+				},
+			},
+			want: want{
+				wantRet: [][]float32{
+					{
+						1, 2, 3,
+					},
+					{
+						4, 5, 6,
+					},
+					{
+						7, 8, 9,
+					},
+				},
+			},
+		},
+		{
+			name: "convert 0 value vector",
+			args: args{
+				vectors: [][]uint8{
+					{
+						0, 0, 0,
+					},
+				},
+			},
+			want: want{
+				wantRet: [][]float32{
+					{
+						0, 0, 0,
+					},
+				},
+			},
+		},
+		{
+			name: "convert max value vector",
+			args: args{
+				vectors: [][]uint8{
+					{
+						math.MaxUint8, math.MaxUint8, math.MaxUint8,
+					},
+				},
+			},
+			want: want{
+				wantRet: [][]float32{
+					{
+						math.MaxUint8, math.MaxUint8, math.MaxUint8,
+					},
+				},
+			},
+		},
+	}
+
+	for _, tc := range tests {
+		test := tc
+		t.Run(test.name, func(tt *testing.T) {
+			tt.Parallel()
+			defer goleak.VerifyNone(tt, goleak.IgnoreCurrent())
+			if test.beforeFunc != nil {
+				test.beforeFunc(test.args)
+			}
+			if test.afterFunc != nil {
+				defer test.afterFunc(test.args)
+			}
+			checkFunc := test.checkFunc
+			if test.checkFunc == nil {
+				checkFunc = defaultCheckFunc
+			}
+
+			gotRet := ConvertVectorsUint8ToFloat32(test.args.vectors)
+			if err := checkFunc(test.want, gotRet); err != nil {
+				tt.Errorf("error = %v", err)
+			}
+		})
+	}
+}
```

Comment:
My comment was min/max value about functions' parameter

Reply:
sorry but what do you mean by `function parameter`? can you please give me an example?

Reply:
e.g., num, dim

Reply:
fixed. 
As we discussed, for the max value cases, I have added code comment